lib/ModelMixins/TableMixin.ts

```diff
@@ -465,6 +489,11 @@ function TableMixin<T extends Constructor<Model<TableTraits>>>(Base: T) {
       return this.activeTableStyle.timeTraits.isSampled;
     }
 
+    @computed
+    get disableDateTimeSelector() {
+      return this.mapItems.length === 0;
+    }
+
```

Comment:
Having date selector enabled for charts doesn't make sense

Reply:
Does this belong in TableMixin or in ChartableMixin?